library/sensu_check_dict.py

```diff
@@ -44,7 +46,11 @@ def main():
                     module.params['name']: module.params['check']
                 }
             })
-
+            
+            command = 'if ip a | grep ' +  module.params['vip'] + ' > /dev/null 2>&1; then ' 
```

Comment:
Can we make the initial command only have the IP check if is is required ?
Most of our checks don't require IP constraint.
Maybe something like this?

```
command = check['checks'][module.params['name']]['command']
if module.params['only_on_ip']:
    command = 'if ip a | grep ' +  module.params['only_on_ip'] + ' > /dev/null 2>&1; then ' +
    command + '; fi'
```